packages/o-spreadsheet-engine/src/plugins/ui_core_views/cell_evaluation/evaluation_plugin.ts

```diff
@@ -159,6 +161,8 @@ export class EvaluationPlugin extends CoreViewPlugin {
     "getSpreadZone",
     "getArrayFormulaSpreadingOn",
     "isEmpty",
+    "getPivotCache",
```

Comment:
`getPivotCache` doesn't mean anything from the business POV.
If the data is cached or not is an implementation detail and the name doesn't tell what's inside this "cache"